com.microsoft.Teams.desktop

```diff
@@ -2,12 +2,14 @@
 Version=1.0
 Type=Application
 Name=Microsoft Teams - Preview
-Comment=Microsoft Teams for Linux is your chat-centered workspace in Office 365.
+Comment=Microsoft Teams is your hub for teamwork in Office 365
+Comment[pt_BR]=O Microsoft Teams é o hub de trabalho em equipe do Office 365
+Comment[es]=Microsoft Teams es el concentrador de trabajo en equipo de Office 365
 Exec=teams %U
 Icon=com.microsoft.Teams
 Terminal=false
 StartupNotify=true
 StartupWMClass=Microsoft Teams - Preview
-Categories=AudioVideo;Network;
+Categories=AudioVideo;Network;Chat;Office;
```

Comment:
Please revert this changes. This `.desktop` has been extracted from the official `.deb` package. I would prefer not diverge too much from upstream.